test/testcafe/framework/page-objects/DeviceChallengePollPageObject.js

```diff
@@ -75,27 +75,41 @@ export default class DeviceChallengePollViewPageObject extends BasePageObject {
   }
 
   getPrimaryButtonText() {
-    return this.body.find('[data-se="o-form-fieldset-container"] .button-primary').innerText;
+    if(userVariables.v3) {
+      return this.form.getButton('Open Okta Verify').innerText;
+    }
+    return this.form.getLink('Open Okta Verify').innerText;
   }
 
   waitForPrimaryButtonAfterSpinner() {
+    if (userVariables.v3) {
+      return this.form.getButton('Open Okta Verify');
+    }
     return Selector('[data-se="o-form-fieldset-container"] .button-primary', { timeout: 4500 });
   }
 
   async clickCancelAndGoBackLink() {
     if (userVariables.v3) {
-      await this.t.click(this.getCancelLink());
+      await this.t.click(this.getFooterCancelPollingLink());
     } else {
       await this.t.click(Selector('a[data-se="cancel-authenticator-challenge"]'));
     }
   }
 
   async clickUniversalLink() {
-    await this.t.click(Selector('.ul-button'));
+    if(userVariables.v3) {
+      await this.form.clickButton('Open Okta Verify');
+    } else {
+      await this.t.click(this.form.getLink('Open Okta Verify'));
+    }
   }
 
   async clickAppLink() {
-    await this.t.click(Selector('.al-button'));
+    if(userVariables.v3) {
+      await this.form.clickButton('Open Okta Verify');
+    } else {
+      await this.t.click(this.form.getLink('Open Okta Verify'));
+    }
   }
```

Comment:
not required, and up to you, but we could combine these two into the `clickLaunchOktaVerifyButton` method below by updating line 119 with the `await this.t.click(this.form.getLink('Open Okta Verify'));` code you used here.